Classes/Controller/MailqueueModuleController.php

```diff
@@ -43,6 +44,7 @@
 final class MailqueueModuleController
 {
     use Traits\TranslatableTrait;
+    use AllowedMethodsTrait;
```

Comment:
The `TYPO3\CMS\Core` namespace is already imported, thus the import should be written like follows:

```suggestion
    use Core\Http\AllowedMethodsTrait;
```